README.md

```diff
@@ -6,20 +6,29 @@ A set of tools for developing [Fathom](http://mozilla.github.io/fathom/) ruleset
 
 Here is an example of how you might use FathomFox to develop a Fathom ruleset that finds the semi-transparent overlays behind in-page pop-ups:
 
-1. Fork, clone, and install the [Fathon-Trainees](https://github.com/mozilla/fathom-trainees/) repository as per its instructions.
-2. Install dependencies and build with `yarn build`, then run `yarn browser` to launch Firefox. (`yarn install` (or just `yarn`) will fail with an error about an incompatible node version, so be sure to use `yarn build` to install modules, or pass the `--ignore-engines` switch to `yarn install`.)
-3. Using `about:debugging`, install your fathom-trainees extension by selecting its `addon/manifest.json` file.
-4. Install [Fathom-Fox](https://addons.mozilla.org/firefox/addon/fathomfox/).
-5. Navigate to a web page that has what you're trying to recognize: a background overlay behind a modal dialog.
-6. Using Firefox's developer tools inspector, select the overlay element.
-7. Switch to the Fathom developer tools tab, enter "overlay" in the label field, and click Save Page.
-8. After you have labeled about 20 pages like this, click the FathomFox icon in the main toolbar, and choose Vectorizer.
-9. Use the Vectorizer to mash down your labeled pages into vectors of floats and save them as a JSON file.
-10. [Use the commandline trainer](https://mozilla.github.io/fathom/training.html#running-the-trainer) to imbibe the vectors and come up with an optimal set of coefficients.
+### Get FathomFox Running
 
-## Where Your Ruleset Goes
+1. Clone the [FathomFox repository](https://github.com/mozilla/fathom-fox/). (This is needed to author rulesets. If you wish to merely collect corpus, you may install FathomFox from [addons.mozilla.org](https://addons.mozilla.org/en-US/firefox/addon/fathomfox/).)
+2. From within the checkout, install dependencies: `yarn run build`.
+3. Run a clean copy of Firefox with FathomFox installed: `yarn run browser`.
+4. Run `yarn run watch` in a separate terminal. This will keep your running copy of FathomFox up to date as you edit your ruleset.
 
-So that you don't have to fork FathomFox itself, the ruleset you're developing goes into a second web extension that you author, the so-called "trainee" extension. An example is [Fathom Trainees](https://github.com/mozilla/fathom-trainees), which you can fork. See its readme for more.
+### Collect a Corpus
+
+1. Navigate to a web page that has what you're trying to recognize: a background overlay behind a modal dialog.
```

Comment:
> I followed the instructions under Quick Start > Collect a Corpus, but the UI appears to be broken

When I first read this, I said "Ah, I forgot to test that!" and assumed I removed a message handler that was needed. However, it stubbornly insists on working on my box! I deleted my node_modules folder and started from scratch. I tried the page you tried (and others). I made sure I'm on the latest Release Firefox (71). Can you help me break it?

Reply:
Hm... Yep this is working for me too now in both Release 71 and Nightly 73.

**In Release 71**
1. `rm -rf node_modules/`
2. Follow instructions in "Get FathomFox Running" in the README
  * `yarn run build`
  *  Replace `yarn run browser` with `web-ext run -s addon/ -f firefox`, since my default browser is Nightly.
3. Follow instructions in the "Collect a Corpus" section in the README
  * Use the [same page](https://www.nytimes.com/2019/12/20/opinion/sunday/democrats-impeachment.html?action=click&module=Opinion&pgtype=Homepage) as before.

**In Nightly 73**
1. Repeat above but just use `yarn run browser`.